locked-asset/token-unstake/src/fees_accumulation.rs

```diff
@@ -1,12 +1,8 @@
 elrond_wasm::imports!();
```

Comment:
should be renamed as it does not accumulate fee's anymore.

Reply:
when is depositFees called ? Isn't fees taken when user makes the unBond ? are not the tokens already inside this contract ?

Reply:
1. Renamed to FeesHandler.
2. depositFees function is called when you reduceLockPeriod. The other case that you may be thinking of, is when the user completely unlocks his tokens (unlock_with_penalty), and in this case, yes, it only applies when the user unbonds the tokens.